work-process.md

```diff
@@ -147,6 +147,12 @@ A few examples of excellent Pull Requests:
 In repositories where deployment involves database migrations and build verifications, we use a
 `PULL_REQUEST.md` template to provide delivery verification steps.
 
+### Continual Improvement
+
+Big cleanups (of code, documentation, or anything else) happen rarely. They are hard to organize, and hard to motivate for. Instead of doing big cleanups every few months (years?) when pain levels are high enough, we prefer to do constant tiny improvements with an approach called [Continual Improvement](https://en.wikipedia.org/wiki/Continual_improvement_process). 
+
+In Niteo, this means that every Pull Request (except those meant for the Priority Lane) should include at least one minor cleanup commit next to the main commit. This can be a typo fix, a description update or a removal of something that no longer makes sense. It does not need to be related to the main commit, it can be any improvement for anything in the repo, however small. 
```

Comment:
Did you leave out the word docstrings to be more general?

I would suggest these changes:
```
For Niteo, this means that every Pull Request (except Priority Lane issues) should include at least one minor cleanup commit included with the main PR commit. It does not need to be related to the main commit in the PR, it can be any improvement for anything in the repo, however trivial. This can include documentation updates, typo fixes, or removal of something that no longer makes sense.
```

Reply:
Yes.

I agree with proposed changes, please apply and rebase.